.github/ISSUE_TEMPLATE/proposal--lightning-talk.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Talk proposal - Lightning Talk (~10 min)
+about: "Propose a short talk that is about 10 minutes in length "
+title: "Talk proposal: <Talk name>"
+labels: Lightning
+assignees: ""
+---
+
+# Title of talk
+
+A short description of the talk. The goal should be to explain to people WHY this topic is interesting and HOW it will help them learn decentralized web protocols and tools. 1-2 paragraphs is fine.
+
+Include a short bio--anything that gives us an idea of why you're interested to speak on this subject. Feel free to include any relevant links.
+
+- Name
+- [Twitter]()
+- [Github]()
+- [Website]()
+
+* [ ] I've read and agree to the [ProtoSchool Seattle Code of Conduct](https://github.com/jimcal/seattle/blob/master/CODE_OF_CONDUCT.md)
```

Comment:
```suggestion
* [ ] I've read and agree to the [ProtoSchool Seattle Code of Conduct](https://github.com/jimcal/seattle/blob/master/CODE_OF_CONDUCT.md)
* [ ] I'm new to giving talks and want to practice present to an organizer over a video call first
```

Reply:
Yes, I should put it in. And also just now I realized I paste the code of conduct from my own fork. Will fix this.